tests/DCPS/Compiler/key_annotation/main.cpp

```diff
@@ -84,6 +86,9 @@ int ACE_TMAIN(int, ACE_TCHAR**)
   failed |= assert_key_count<KeyedUnion>(1);
   failed |= assert_key_count<KeyedUnionStruct>(2);
   failed |= assert_key_count<MultidimensionalArrayStruct>(2 * 3 + 2 * 3 * 4);
+  failed |= assert_key_count<ImpliedKeys::StructA>(5);
```

Comment:
Could this check for the specific keys in addition to the number of keys?

Reply:
It could and, in fact, I would like it to. I just haven't gotten around to it.